<launch>
    <arg name="forkpos" default="/forklift_pose"/>
    <!-- <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="-0.16 0 0.58 3.14 0.43633231 0 base_link camera_link 100" /> -->

    <!--megapose-->
    <include file="$(find visp_megapose)/launch/megapose_client_oilpalm.launch"/>

    <node pkg="forklift_server" type="PBVS_server_differential.py" name="PBVS_server" output="screen">
        <!--Subscriber Topic setting/-->
        <param name="odom" value="/wheel_odom" /><!--里程計 Topic（map -> base_link, 是tf相對關係, 非輪式里程計）/-->
        <param name="pose_topic" value="/oilpalm" /><!--相機對位的 Topic/-->
        <param name="object_filter" value="False" /><!--(True: , False: )/-->
        <param name="confidence_minimum" value="0.7" /><!--megapose信心值低於它停止動作/-->

        <!--camera parking setting-->
        <param name="camera_tag_offset_x" type="double" value = "0.0" /><!--對位目標點與tag的左右偏移量/-->
        <param name="camera_desired_dist_threshold" type="double" value = "0.35" />  <!--距離目標點的直線距離小於threshold，就不執行move_nearby_parking_lot動作-->
        <param name="camera_horizon_alignment_threshold" type="double" value = "0.03" />  <!--車體相機水平對齊偏差容許距離（公尺）/-->

    </node>
</launch>
